pkg/kubelet/cm/devicemanager/topology_hints_test.go

```diff
@@ -32,13 +32,18 @@ import (
 )
 
 type mockAffinityStore struct {
-	hint topologymanager.TopologyHint
+	hint   topologymanager.TopologyHint
+	policy topologymanager.Policy
 }
 
 func (m *mockAffinityStore) GetAffinity(podUID string, containerName string) topologymanager.TopologyHint {
 	return m.hint
 }
 
+func (m *mockAffinityStore) GetPolicy() topologymanager.Policy {
+	return m.policy
+}
+
```

Comment:
Does this ever get exercised anyhwere? It looks like you just always set the policy to `nil` in all test cases.

Reply:
Fixed in latest commit